packages/reputation-miner/ReputationMiner.js

```diff
@@ -642,12 +654,60 @@ class ReputationMiner {
     const [round, index] = await this.getMySubmissionRoundAndIndex();
     const repCycle = await this.getActiveRepCycle();
     const submission = await repCycle.getDisputeRounds(round, index);
-    const targetNode = submission[8].add(submission[9]).div(2);
+    const searchInfo = await repCycle.getDisputeRounds2(round, index);
+    const targetNode = submission[8];
     const targetNodeKey = ReputationMiner.getHexString(targetNode, 64);
 
     const intermediateReputationHash = this.justificationHashes[targetNodeKey].jhLeafValue;
-    const [branchMask, siblings] = await this.justificationTree.getProof(targetNodeKey);
-    return repCycle.respondToBinarySearchForChallenge(round, index, intermediateReputationHash, branchMask, siblings, {
+    const proof = await this.justificationTree.getProof(targetNodeKey);
+    const [branchMask] = proof;
```

Comment:
TIL a little more about JS variable assignment; would've thought it was `[branchMask,]`